### PR TITLE
Postgres adapter

### PR DESCRIPTION
**Pull request by or-else**
Cleanup and a couple of bug fixes.

### CODE REVIEW

File: server/db/postgres/adapter.go [hunk 1]
```diff
@@ -0,0 +1,3543 @@
+//go:build postgres
+// +build postgres
+
+// Package postgres is a database adapter for PostgreSQL.
+package postgres
+
+import (
+	//"bytes"
```
Comment (aforge) [inline server/db/postgres/adapter.go:8]: Remove.

**Review by aforge — COMMENTED** [review]
Do you think you can add docker compose configs in [docker](https://github.com/tinode/chat/tree/master/docker)/docker-compose/?

**Comment by or-else** [comment]
> Do you think you can add docker compose configs in [docker](https://github.com/tinode/chat/tree/master/docker)/docker-compose/?

Added the files, but I cannot test them:

1. The files are for amd64, I'm on arm.
2. The `tinode-postgres` container has to be built and published first.

File: server/db/postgres/adapter.go (hunk 1 above)
Comment (or-else) [reply server/db/postgres/adapter.go]: Done

**Review by aforge — APPROVED** [review]

**Pull request merged**